scene/resources/skeleton_profile.cpp

```diff
@@ -390,6 +390,52 @@ bool SkeletonProfile::has_bone(const StringName &p_bone_name) {
 	return is_found;
 }
 
+void SkeletonProfile::init_from_skeleton(Skeleton3D *skeleton) {
```

Comment:
```suggestion
void SkeletonProfile::initialize_from_skeleton(Skeleton3D *skeleton) {
```

We typically spell out abbreviations.